package.json

```diff
@@ -14,7 +14,8 @@
     "postbuild": "yarn fix-bins",
     "watch-build": "npx tsc-watch --project tsconfig.json --onSuccess \"bob build\"",
     "rebuild": "bob build --incremental",
-    "test": "jest --forceExit --no-watchman",
+    "test": "vitest --no-watch",
+    "test:watch": "vitest",
```

Comment:
This is added to be able to run in watch mode:
- Vitest runs in watch mode by default
- So, to run once (i.e. `test` existing behaviour), we add `--no-watch`
- But with `--no-watch`, we cannot do `yarn test -w` because `-w` conflicts and is ignored